app/assets/javascripts/index/note.js

```diff
@@ -32,97 +32,94 @@ OSM.Note = function (map) {
   };
 
   page.load = function (path, id) {
-    initialize(path, id);
-    moveToNote();
-    page.load = function () {
-      // the original page.load content is the function below, and is used when one visits this page, be it first load OR later routing change
-      // below, we wrap "if map.timeslider" so we only try to add the timeslider if we don't already have it
-      function originalLoadFunction() {
-        initialize(moveToNote);
-      }  // end originalLoadFunction
-
-      // "if map.timeslider" only try to add the timeslider if we don't already have it
-      if (map.timeslider) {
-        originalLoadFunction();
-      } else {
-        var params = querystring.parse(location.hash.substring(1));
-        addOpenHistoricalMapTimeSlider(map, params, originalLoadFunction);
-      }
-    };
-
-    function initialize(path, id) {
-      content.find("button[type=submit]").on("click", function (e) {
-        e.preventDefault();
-        var data = $(e.target).data();
-        var form = e.target.form;
-
-        $(form).find("button[type=submit]").prop("disabled", true);
-
-        $.ajax({
-          url: data.url,
-          type: data.method,
-          oauth: true,
-          data: {text: $(form.text).val()},
-          success: function () {
-            OSM.loadSidebarContent(path, function () {
-              initialize(path, id);
-              moveToNote();
-            });
-          },
-          error: function (xhr) {
-            $(form).find("#comment-error")
-              .text(xhr.responseText)
-              .prop("hidden", false);
-            updateButtons(form);
-          }
-        });
-      });
+    // the original page.load content is the function below, and is used when one visits this page, be it first load OR later routing change
+    // below, we wrap "if map.timeslider" so we only try to add the timeslider if we don't already have it
+    function originalLoadFunction(path, id) {
+      initialize(path, id);
+      moveToNote();
+    }  // end originalLoadFunction
+
+    // "if map.timeslider" only try to add the timeslider if we don't already have it
+    if (map.timeslider) {
+      originalLoadFunction();
+    } else {
+      var params = querystring.parse(location.hash.substring(1));
```

Comment:
Much of this diff is just noise from changing the indentation level. I refrained from modernizing any of this code, because it has already been refactored and modernized upstream, so that would just lead to an even messier merge conflict the next time around.